octoprint_mrbeam/templates/mrbeam_initscript.jinja2

```diff
@@ -1,5 +1,7 @@
 <script lang="javascript">
     var BEAMOS_VERSION = "{{ beamosVersionNumber }}";
+    // PLUGINS_VERSIONS is set by single quotes since "pluginVersions" is returning a JSON dump which will break with double quotes
+    var PLUGINS_VERSIONS = '{{ plugins_versions }}';
```

Comment:
Instead of using quotes you could try to use some of flask filters to make it easier. They seem to be pretty powerful.
Here is an example that would match this needs using tojson and safe https://stackoverflow.com/questions/24719592/sending-data-as-json-object-from-python-to-javascript-with-jinja
You would also need to remove the json.dumps call on the initialisation.